quipucords/tests/api/aggregate_report/test_aggregate_report_model.py

```diff
@@ -330,9 +330,11 @@ def test_build_aggregate_report_system_fingerprint(
 ):
     """Test build_aggregate_report using generated Report, SystemFingerprints, etc."""
     report, expected_aggregate_report = report_and_expected_aggregate
-    aggregated: AggregateReport = build_aggregate_report(report.id)
+    aggregated = build_aggregate_report(report.id)
     assert aggregated is not None
-    assert aggregated == expected_aggregate_report
+    assert reformat_aggregate_report_to_dict(
+        aggregated
+    ) == reformat_aggregate_report_to_dict(expected_aggregate_report)
```

Comment:
Now that AggregateReport is its own model I don't see value on the current implementation of the fixture `report_and_expected_aggregate`.

The problem here is that `build_aggregate_report` won't produce a new aggregate report instance. It will simply get the one instance created in that fixture. So in this test if feels we are testing if `1 == 1`.

I would replace the AggregateReport instance returned by that fixture with a dictionary containing all the expected data and in the exact format we expect it to be used.

After that, change this assertion to something like this

```suggestion
    assert reformat_aggregate_report_to_dict(
        aggregated
    ) == expected_aggregate_report
```


Also, if you follow my suggestion about the serializer, this would change to 

```suggestion
    assert AggregateReportSerializer(instance=aggregated).data == expected_aggregate_report
```